### PR TITLE
Issue #13213: Remove '//ok' comments 

### PR DESCRIPTION
**Pull request by Pinti0001**
Issue #13213 
Removed   //  ok  from InputJavadocPackageHtmlIgnored2.java   and   InputJavadocParagraphCorrect.java with its  suppressions

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -942,12 +942,6 @@
             files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValidWithTab.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValidWithTab.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocpackage[\\/]pkghtml[\\/]InputJavadocPackageHtmlIgnored2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocparagraph[\\/]InputJavadocParagraphCorrect.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocparagraph[\\/]InputJavadocParagraphCorrect.java"/>
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:950]: The changes lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
Please look at the failing CI : 

```
[ERROR] Failures: 
[ERROR]   CommitValidationTest.testCommitMessageHasProperStructure:216 Commit 5a5c99b0620e0946ffcc964766c287489e818e32 message: "Update InputUnnecessaryParenthesesIfStatement2.java\n\nAccidentally clipped off brace added back" is invalid
Proper commit message should adhere to the following rules:
    1) Must match one of the following patterns:
        ^Issue #\d+: .*$
        ^Pull #\d+: .*$
        ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    2) It contains only one line of text
    3) Must not end with a period, space, or tab
    4) Commit message should be less than or equal to 200 characters

The rule broken was: 1

```
Checkstyle requires your commit message to adhere to certain rules. Please modify accordingly.
Make sure to test your build using `mvn clean verify` before pushing changes.

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (Pinti0001) [reply config/checkstyle-input-suppressions.xml:950]: Hello @MANISH-K-07  thanks for replying .
when I run command `mvn clean verify` then it build successfully so why my test cases failed can you suggest me.

Comment (MANISH-K-07) [reply config/checkstyle-input-suppressions.xml:950]: As mentioned above, the failing CI are due to bad format of commit message.
Modifying it according to checkstyle rules should make CI happy..

Comment (MANISH-K-07) [reply config/checkstyle-input-suppressions.xml:950]: Please don't close and reopen PRs. Use existing PR itself. All changes can be made through git cmd

File: config/checkstyle-input-suppressions.xml [hunk 2]
```diff
@@ -942,12 +942,6 @@
             files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValidWithTab.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValidWithTab.java"/>
-  <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:945]: You still have failing CI : 
https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=20049&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111&l=1331

**Review by MANISH-K-07 — COMMENTED** [review]
Please ensure that `mvn clean verify` passes on local before pushing changes up

File: config/checkstyle-input-suppressions.xml (hunk 2 above)
Comment (Pinti0001) [reply config/checkstyle-input-suppressions.xml:945]: Now check again @MANISH-K-07

File: config/checkstyle-input-suppressions.xml [hunk 3]
```diff
@@ -942,12 +942,6 @@
...
-            files="checks[\\/]javadoc[\\/]javadocpackage[\\/]pkghtml[\\/]InputJavadocPackageHtmlIgnored2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocparagraph[\\/]InputJavadocParagraphCorrect.java"/>
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:948]: @Pinti0001 ,
I'm afraid you might have conflicting files with PR #14563 .
This is why it is recommended to drop a comment like "I am on xxxxModule" on the issue page to let other contributors know that the module is being worked on....

**Review by MANISH-K-07 — COMMENTED** [review]
@romani , @nrmancuso 

`FYI : `This PR might raise conflicts with PR #14563 while merge.

**Review by romani — APPROVED** [review]
Ok to merge if CI pass

**Comment by romani** [comment]
Thanks a lot for first contribution! Ply do more

**Pull request merged**